clients/cobol-lsp-vscode-extension/tslint.json

```diff
@@ -31,7 +31,8 @@
         "arrow-parens": [
             true,
             "ban-single-arg-parens"
-        ]
+        ],
+        "prettier": true
```

Comment:
Why do we store tslint config on the repository?

Reply:
To have it nice for everyone.

Reply:
Isn't tslint a dead project?

Reply:
It is. But I believe that eslint understand tslint configuration.

Reply:
I think that it deserves an additional user story to provide a proper eslint config for our requirements.

Reply:
Also, the story should be about switching to eslint. I agree.